Core/Importers/UkrSibTransactionsImporter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using ExcelDataReader;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading;
+
```

Comment:
Codacy found an issue: [Refactor this method to use a generic collection designed for inheritance.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)